src/main/kotlin/io/github/kotools/csv/reader/Process.kt

```diff
@@ -32,6 +35,13 @@ private infix fun <T : Any> List<T>.getPage(pagination: Reader.Pagination):
         List<T> = chunked(pagination.size)
     .getOrElse(pagination.page - 1) { emptyList() }
 
+private fun <V : Any> Map<String, V>.withoutBlankKeys():
+        Map<NotBlankString, V> = try {
+    mapKeys { it.key.notBlank }
```

Comment:
We can use `it.key.notBlankOrNull ?: error(...)` instead of using a `try...catch` block.